src/components/Login.js

```diff
@@ -92,31 +98,66 @@ const Login = () => {
 								</Alert>
 							</Center>
 						)}
-						<Box>
-							<Input
-								name="facultyNumber"
-								placeholder="Факултетен номер"
-								required
-								onChange={validateFacultyNumber}
-								isInvalid={isInvalid}
-								errorBorderColor="red.300"
-							/>
-						</Box>
-						<Box>
-							<Input
-								name="password"
-								placeholder="Парола"
-								type="password"
-								required
-								onChange={handleChange}
-							/>
-						</Box>
-						<Button isDisabled={isInvalid} type="submit">
-							Влез
-						</Button>
-					</VStack>
+					</Center>
+					<Stack spacing={2} margin="1rem" marginBottom="5rem">
+						<Center>
+							<Box>
+								<FormControl isRequired>
+									<Input
+										id="id_facultyNUmber"
+										name="facultyNumber"
+										placeholder="Потребителско име"
+										required
+										onChange={validateFacultyNumber}
+										bg="white"
+										maxWidth="30rem"
```

Comment:
When I start inputting my username (numbers only), I immediately get an error while typing the first number. Can you tweak the validation so that:

- if I input a different than number char - the validation error message appears immediately
- if I start typing my faculty number - the validation does not fire while I input only numbers